docs/user-guide/host-settings.md

```diff
@@ -0,0 +1,193 @@
+the Linux distribution for guides on how to persist the settings in for example
+[Netplan](https://documentation.ubuntu.com/server/explanation/networking/configuring-networks/index.html) or 
+[Network Manager](https://networkmanager.dev/docs/admins/). For additional
+details consult the [kernel bonding
+driver](https://www.kernel.org/doc/html/latest/networking/bonding.html)
+documentation.
+
+## MCLAG / ESLAG
+TODO - does this active/active?
+
+The multi-chassis LAG architecture is a way to reduce the number of failure points
+in a network architecture. At the physical layer an MCLAG topology is a single
+server connected to two different switches and those switches are connected.
+For a total of 3 physical connections.
+
+ESLAG is a similar topology as MCLAG with the beneficial difference that the
+switches do not need a to be connected to each other. For a total of 2 physical
+connections
+
+The standard that governs this technology is commonly referred to as 802.3ad.
+
+### Server Settings
+
+The supported server side setting for bonding is `802.3ad`, with hashing policy
+`layer2` or `layer2+3`. To configure this in linux:
+``` bash
+sudo ip link add $bond_name type bond miimon 100 mode 802.3ad layer2
+sudo ip link set $slave1_name master $bond_name
+sudo ip link set $slave2_name master $bond_name
+```
+
+The `miimon 100` indicates that the  bonding driver will use miimon to
+determine if the link is `up` and it will poll every 100 milliseconds.
+
+`layer2` or `layer2+3` is the hashing mode, which controls the input into the
+hash function. See the [kernel bonding
+driver](https://www.kernel.org/doc/html/latest/networking/bonding.html)
+documentation for additional details.
+
+After the bond has been created, it can receive a dhcp address, or be
+statically assigned:
+
+`sudo ip addr add 10.10.10.3/24 dev bond0`
+
+To check the status of this bond:
+
+`cat /proc/net/bonding/bond0`
+
+
+## LAG, Bonding, Teaming, Bundling
+
+A LAG configuration can be used to increase the bandwidth to and from an end
+host. For example, if a server has 2 25 Gbps ports, the ports can be bonded so
+they will have an aggregate 50 Gbps of available bandwidth. At the physical
+layer there are two cables going from the server to a single switch.
+
+To enable fast detection of faults, `802.3ad` is the recommended mode for bonds
+attaching to the Open Network Fabric. The steps to configure the bond are the
+same as the [above server settings](#server-settigns)
+
+The kernel bonding driver discusses other modes for bonding, several of those modes don't require cooperation on the
+switch side, so there is no fabric configuration needed. The primary concern
+for using these modes is out of order TCP delivery and fault detection and
+correction time.
+
+## Active / Passive Fault Tolerance
+
+Some networking equipment doesn't support MCLAG or ESLAG. If a user would like
+the host to have a fault tolerant availability the Linux server bond can be
```

Comment:
I think it's good to explicitly mention that a pair of Unbundled connections to be used for it

Reply:
Agreed.